.github/workflows/tests.yml

```
@@ -128,31 +128,21 @@ jobs:
           vendor/bin/testbench migrate --ansi
           vendor/bin/testbench package:test --ansi --coverage
 
-      - name: Generate reports for CodeClimate
-        id: reports
-        if: false # TODO: after #197
+      - name: Generate reports for Coveralls
+        if: needs.prepare.outputs.has-coveralls == '1'
```

actionlint (1.7.12)

[error] property "prepare" is not defined in object type {configs: {outputs: {string => string}; result: string}; upstream: {outputs: {has-changes: string}; result: string}}

(expression)